src/gravel/controllers/orch/ceph.py

```diff
@@ -234,21 +234,62 @@ def _get_ruleset_id(self, name: str) -> int:
             logger.exception(e)
             raise NoRulesetError()
 
-    def _set_default_ruleset_config(self, rulesetid: int) -> bool:
+    def config_set(self, name: str, value: str, who: str) -> bool:
```

Comment:
```suggestion
    def config_set(self, who: str, name: str, value: str) -> bool:
```
 (nit) but order these according to the mon command?